src/templates/index.html

```diff
@@ -0,0 +1,136 @@
+                        </div>
+                        <div class="col-auto">
+                            <i class="fas fa-random fa-2x text-gray-300"></i>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <!-- Entry Count Card -->
+        <div class="col-md-3 mb-3">
+            <div class="card border-left-primary shadow h-100 py-2">
+                <div class="card-body">
+                    <div class="row no-gutters align-items-center">
+                        <div class="col mr-2">
+                            <div class="text-xs font-weight-bold text-primary text-uppercase mb-1">Total Variants
+                            </div>
+                            <div class="h5 mb-0 font-weight-bold text-gray-800">{{ t }}</div>
+                        </div>
+                        <div class="col-auto">
+                            <i class="fas fa-search fa-2x text-gray-300"></i>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </div>
+
+    <!-- Content Row -->
+    <div class="row">
+        <div class="col-6">
+            <div class="card mt-3">
+                <div class="card-header">
+                    Links
+                </div>
+                <div class="card-body">
+                    <div class="list-group">
+                        <a class="list-group-item list-group-item-action" href="http://grch37.ensembl.org"
+                            target="_blank">Ensembl
+                            GRCh38</a>
+                        <a class="list-group-item list-group-item-action" href="http://ensembl.org"
+                            target="_blank">Ensembl
+                            GRCh37</a>
+                        <a class="list-group-item list-group-item-action" href="https://omim.org"
+                            target="_blank">OMIM®</a>
+                        <a class="list-group-item list-group-item-action" href="https://ncbi.nlm.nih.gov/snp"
+                            target="_blank">dbSNP</a>
+                        <a class="list-group-item list-group-item-action" href="https://ncbi.nlm.nih.gov/nuccore"
+                            target="_blank">GenBank</a>
+                        <a class="list-group-item list-group-item-action" href="https://genecards.org/"
+                            target="_blank">GeneCard</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<!-- /.container-fluid -->
+
+</div>
+<!-- End of Main Content -->
+
+<!-- Footer -->
+<footer class="sticky-footer bg-white">
+    <div class="container my-auto">
+        <div class="copyright text-center my-auto">
+            <span>Copyright &copy; UMC Utrecht 2020</span>
+        </div>
+    </div>
+</footer>
+<!-- End of Footer -->
+
+
+{% endblock %}
```

Comment:
I see a warning: "No newline at end of file".

Reply:
not needed for HTML